apps/src/applab/designMode.js

```diff
@@ -925,24 +925,27 @@ designMode.createScreen = function () {
 /**
  * Changes the active screen by triggering a 'CHANGE_SCREEN' action on the
  * Redux store.  This change propagates across the app, updates the state of
- * React-rendered components, and eventually calls onScreenChange, below.
+ * React-rendered components, and eventually calls renderScreens, below.
  * @param {!string} screenId
  */
 designMode.changeScreen = function (screenId) {
   studioApp.reduxStore.dispatch(screens.changeScreen(screenId));
 };
 
 /**
- * Responds to changing the active screen by toggling all screens to be
- * non-visible, unless they match the provided screenId, and opens the element
+ * Given the currentScreenId in our redux store, toggles all screens to be
+ * non-visible, unless they match the provided screenId, with the element
  * property editor for the new screen.
  *
  * This method is called in response to a change in the application state.  If
  * you want to change the current screen, call designMode.changeScreen instead.
  *
  * @param {!string} screenId
  */
-function onScreenChange(screenId) {
+function renderScreens(screenId) {
```

Comment:
Motivation here was to make this method seem less like a "do this thing when we see a change in this state" and more like a "make the DOM reflect the current state of the store", i.e. I want it to be more declarative than imperative. Not sure whether there might still be a better name. Arguably I should also make a similar sort of change for onInterfaceModeChange - maybe to `renderInterfaceMode`?